components/ContentTemplates/HeadingsTemplate.tsx

```diff
@@ -1,5 +1,155 @@
+					title="Why are HTML Headings Important?">
+					<p>
+						Headings are important for a number of reasons. <br></br>
+						<br></br>
+					</p>
+					<ul>
+						<li>
+							They give order and structure to the web page, making it easier
+							for users to understand the layout and content of the page.{" "}
+						</li>
+						<br></br>
+						<li>
+							They allow the browser to index and structure the page visually,
+							which helps users find what they are looking for.{" "}
+						</li>
+						<br></br>
+						<li>
+							They allow screen readers and users to skip to specific sections
+							of the page, so they can find the information they need quickly.
+						</li>
+					</ul>
+				</TemplateSection>
+				<TemplateSection
+					sectionName="includedInHeadings"
+					title="What Should Be Included in a Heading?">
+					<p>
+						When using headings, it is important to include descriptive titles
+						and subheadings that accurately describe the content on the page.
+						This will help users to find what they are looking for more easily.
+						Headings should also be concise and to the point. They should
+						clearly summarize the content that follows.
+					</p>
+				</TemplateSection>
+
+				<TemplateSection
+					sectionName="goodHeadings"
+					title="How to Write a Good Heading">
+					<p>
+						Writing a good heading is all about being descriptive, and
+						remembering how to nest heading sections within one another in
+						chronological order.
+					</p>
+					<p>Below is a code snippet example with semantic section headings.</p>
+					<CodeBlock
+						codeSnippet={`<h1> Yoga for Developers </h1>
+	<h2> What is Yoga? </h2>
+		<h3> The History of Yoga </h3>
+			<h4> Yoga's Origin </4>
+			<h4> Yoga in Modern Times </h4>
+	<h2> The Benefit of Yoga </h2>
+		<h3> Strength Conditioning </h3>
+			<h4> Power Yoga </4>
+		<h3> Increasing Flexibility</h3>
+			<h4> Gentle Stretches </4>
+				<h5> Stretches for the hands </h5>
+					<h6> Prayer Pose </h6>
+					<h6> Nerve Gliding Exercises </h6>
+						`}
+					/>
+				</TemplateSection>
+
+				<TemplateSection
+					sectionName="avoidHeadings"
+					title="What Should We Avoid When Using Headings?">
+					<ul>
+						<li>
+							Avoid using more than one H1 on a page, as this can confuse the
+							structure and hierarchy of the page. Instead, the H1 should be
+							used for the most important topic on the page.
+						</li>{" "}
+						<br></br>
+						<li>
+							Headings should not be chosen for style purposes - use CSS to
+							style sections instead.{" "}
+						</li>
+						<br></br>
+						<li>
+							Headings should not be placed in non sequential order to ensure
+							that the structure of the page makes sense.
+						</li>
+						<br></br>
+					</ul>
+				</TemplateSection>
+				<TemplateSection sectionName="WCAGCriteria" title="WCAG Criteria">
+					<a
+						href="w3.org/WAI/WCAG21/Understanding/section-headings.html"
```

Comment:
To keep WCAG links consistent

```suggestion
						href="https://www.w3.org/TR/WCAG21/#section-headings"
```

Reply:
resolved